dev/Dockerfile-dev

```diff
@@ -0,0 +1,18 @@
+FROM python:3.8.1-alpine3.11
+
+RUN apk add --no-cache gcc musl-dev linux-headers make curl git
+
+WORKDIR /srv/code
+COPY requirements.txt .
+COPY setup.cfg .
+COPY Makefile Makefile
```

Comment:
Mmh, I'm not a huge fan of this since many of the commands don't work any more. As said on the phone, we don't really need it in the container, either (since we have the `cleanup.sh`). Is there a specific reason you want the Makefile available in the container? If not, I would suggest to remove it. If yes, Maybe you can split the Makefile into the parts that are functional inside the container and those who are not? Maybe have a look at [include](https://www.gnu.org/software/make/manual/html_node/Include.html), I think with this it should be possible.

Reply:
I understand you idea but I strongly recommand to provide the Makefile inside the dev environment so everybody can use `make black` or `make run-debug`. Of course some commands won`t work but I can add a hint in a comment in the Makefile. These commands do not harm there. Splitting up is less explicit so I prefer not to do so.